src/AuthenticationPortal.Contracts/Errors/CustomException.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace AuthenticationPortal.Contracts
+{
+    public class CustomException : BaseException
+    {
+        public CustomException(int code) : base(CustomErrorCodes.getErrorMessage(code), code)
```

Comment:
what does this class do

Reply:
So, whenever we need to give custom error response to the user, the api will throw a new exception Like
throw new CustomException(789);
where 789 can be a custom error code of our own which will have a appropriate error message associated with it, so this error message and custom error code will be sent in the response

Reply:
Done